### DISCUSSION TITLE
Welcome to MARC2RDA Discussions!

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Make plans for discrete pieces of work which may be converted into issues
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪

Please see the

**Comment by gerontakos** [comment]
Wow we have a discussion board. All In with Github! I think we'll find it integrates well with other MS products, with which we're also All In.
